third-party/llvm/Makefile

```diff
@@ -73,55 +67,25 @@ $(LLVM_CONFIGURED_HEADER_FILE): $(LLVM_SRC_FILE)
 	    -Wno-dev \
 	    $(LLVM_SRC_DIR) ; \
 	else \
-	  cd $(LLVM_BUILD_DIR) && $(LLVM_SRC_DIR)/configure \
-	     CC='$(CC)' CXX='$(CXX)' \
-	     LDFLAGS='$(LDFLAGS)' --prefix=$(LLVM_INSTALL_DIR) \
-	     --enable-optimized \
-	     --disable-clang-arcmt \
-	     --disable-clang-static-analyzer \
-	     --disable-clang-rewriter \
-	     --disable-docs \
-	     --disable-zlib \
-	     --enable-targets=host,x86,x86_64,aarch64 $(CHPL_LLVM_CONFIGURE_DEBUG) ; \
-	  cd $(LLVM_BUILD_DIR) && make clean ; \
+	  echo Error: LLVM requres cmake 3.4.3 or later to build; \
+	  exit 1; \
 	fi
-# note: automake llvm builds don't create nested Makefiles
-# that is what the make clean above accomplishes
 
 
 $(LLVM_HEADER_FILE):
-	if [ -f $(LLVM_CONFIGURE_FILE) ]; then \
-	  cd $(LLVM_BUILD_DIR) && make install-local ; \
-	else \
-	  cd $(LLVM_BUILD_DIR) && cmake --build . --target install-llvm-headers ; \
-	fi
+	cd $(LLVM_BUILD_DIR) && cmake --build . --target install-llvm-headers
 
 # note: the conditional below allows parallel make to continue
 # if cmake us also using Make.
 $(LLVM_SUPPORT_FILE):
-	if [ -f $(LLVM_CONFIGURE_FILE) ]; then \
-	  cd $(LLVM_BUILD_DIR) && $(MAKE) install-cmake-exports ; \
-	  cd $(LLVM_BUILD_DIR)/lib/Support && $(MAKE) ; \
-	  cd $(LLVM_BUILD_DIR)/lib/Support && $(MAKE) install ; \
-	elif [ -f $(LLVM_BUILD_DIR)/Makefile ]; then \
```

Comment:
AFAIK we want this 2nd element of the conditinoal to allow parallel make (make -j 8 say) to work for building LLVMSupport but you might have a different solution to that problem.

Reply:
Thanks.  That was an oversight.